api/controllers/FlairController.js

```diff
@@ -161,14 +144,14 @@ module.exports = {
 
     Event.find(appData).exec(function (err, events) {
       if (err) {
-        res.json({error: "Unknown"}, 500);
+        res.status(500).json({error: "Unknown error"});
       }
       var now = moment();
       if (events.length) {
         var then = moment(events[0].createdAt);
         then.add(2, 'minutes');
         if (then.isAfter(now)) {
-          return res.json({error: "You set your flair too recently, please try again in a few minutes."}, 400);
+          return res.status(400).json({error: "You set your flair too recently, please try again in a few minutes."});
```

Comment:
All the sails-implemented error messages omit their error messages when running on production, to avoid leaking sensitive info to the client. We should use them when we're not sure what the error message is going to be (e.g. sails.badRequest(err)), but we shouldn't use them if we have prewritten error messages that we want the client to see.